cmake/packaging.cmake

```diff
@@ -250,12 +252,23 @@ configure_file(cmake/CPackConfig.cmake.in package/CPackConfig.cmake @ONLY)
 set(CPACK_GENERATOR "${PACKAGING_SYSTEM}")
 
 if(DEFINED PLATFORM_LINUX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/")
+  set(CPACK_STRIP_FILES TRUE)
+
+  if(CPACK_GENERATOR STREQUAL "TGZ")
+    set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}_1.linux.x86_64")
```

Comment:
(referenced from above)

This is why `CPACK_PACKAGE_NAME` and `CPACK_PACKAGE_VERSION` are set explicitly. Without setting those above this line fills in blank values for those variables.

Reply:
Ah I see, right, I totally missed that.
Those variables are set after including the CPack cmake file, so if you use the before that, they'll be empty.
It's fine, if we keep CPack in the future we will probably move the logic after CPack is included and will be run each time CPack is called (like for what's happening right now with the CPackConfig.cmake.in file).